src/main/java/com/checkmarx/flow/service/SastScanner.java

```diff
@@ -116,7 +117,7 @@ public boolean isEnabled() {
         if (enabledScanners == null || enabledScanners.isEmpty()) {
```

Comment:
it would be nicer if the configuration read happens only once in the 'dispatcher' (FlowService in this case), and pass the value, instead of read it in each scanner

Reply:
In case of SAST the validation is different than the sca implementation since we want to check empty condition to support backwards compatibility.